Huge update

IgnusVam opened the pull request:
- New z level
- New ship, shuffle, etc.
- Improvements to the trade ship
- Extra work on synx, still not finished
- Secrets
- Bunch of other stuff I forgot.. I need to make more commits with big updates like this.

IgnusVam committed:
Merge branch 'master' of https://github.com/CHOMPstation/CHOMPstation

IgnusVam committed:
Merge branch 'master' of https://github.com/CHOMPstation/CHOMPstation

IgnusVam committed:
Huge update!
- New pet for cargo
- Many fixes on the map
- A whole new z-level, shuttle, and casino ship! (not yet fully finished)
- New version of synxes, and a lot of their code has been fixed. They should also mimic now.
- Lots of fixes and additions
- Secrets and easter eggs, including one none of you will get but is blindingly obvious.
- If you're still reading this, have a cookie.

IgnusVam committed:
- Re-comments out centcom jobs until they're fixed. (seriously this needs to be fixed :v )

IgnusVam committed:
Merge branch 'master' of https://github.com/CHOMPstation/CHOMPstation

- Reverts the polaris 1 to a working state.

IgnusVam committed:
Fixes a condimaster. Should fix problems
Removes a duplicated file.

KasparoVy commented on code/modules/mob/living/simple_animal/animals/chungus.dm:
```diff
@@ -0,0 +1,155 @@
+/mob/living/simple_animal/hostile/chungus
+	name = "big chungus"
+	desc = "There's a chungus amongus."
```
Hooooly shit, hahaha.

IgnusVam replied:
Addition by somebody else, wanted me to add it for them.

KasparoVy replied:
Got a good laugh reading through the diff. Nicely done

IgnusVam committed:
- Adds a control button to the casino exit door.. maybe this will fix Travis? I have no idea.

IgnusVam committed:
- Resets a door controller and remakes it, to avoid a faulty tag "change"
- Changes some carpet colors in the casino so it's not the same color.
- Changes the area in a secret.

IgnusVam committed:
- Resets and remakes a door to fix a tag issue.
